test/workspace.test.js

```diff
@@ -0,0 +1,198 @@
+            .withGenerators(APP_GENERATOR_DEPENDENCIES)
+            .withPrompts({
+                app: 'appName'
+            })
+            .inTmpDir(() => {
+                process.chdir(workingDirectory);
+                fse.mkdirSync(`./${appPlugin}`);
+                process.chdir(path.join(workingDirectory, appPlugin));
+            });
+
+        // Run yo phovea:workspace
+        await helpers
+            .run(path.join(__dirname, '../generators/workspace'))
+            .withGenerators(WORKSPACE_GENERATOR_DEPENDENCIES)
+            .inTmpDir(() => {
+                workspace = workingDirectory.replace('/tmp/', '');
+                process.chdir(workingDirectory);
+            });
+    });
+
+    // Switch back the cwd
+    afterAll(() => {
+        process.chdir(cwd);
+    });
+
+    it('generates expected workspace files', () => {
+        assert.file(expectedFiles);
+    });
+
+    it('generates expected <workspace>.iml file', () => {
+        assert.file(`.idea/${workspace}.iml`);
+    });
+
+    it(`adds correct entries in ".idea/vcs.xml"`, () => {
+        const libEntry = `<mapping directory="$PROJECT_DIR$/${libPlugin}" vcs="Git" />`;
+        const appEntry = `<mapping directory="$PROJECT_DIR$/${appPlugin}" vcs="Git" />`;
+        assert.fileContent(`.idea/vcs.xml`, libEntry);
+        assert.fileContent(`.idea/vcs.xml`, appEntry);
+    });
+
+    it(`adds correct entries in ".idea/<workspace>.iml"`, () => {
+        const libEntry = `<sourceFolder url="file://$MODULE_DIR$/${libPlugin}" isTestSource="false" />`;
+        const appEntry = `<sourceFolder url="file://$MODULE_DIR$/${appPlugin}" isTestSource="false" />`;
+        assert.fileContent(`.idea/${workspace}.iml`, libEntry);
+        assert.fileContent(`.idea/${workspace}.iml`, appEntry);
+    });
+
+    it(`imports all workspace plugins in "phovea_registry.js"`, () => {
+        const libEntry = `import '${libPlugin}/phovea_registry.js';`;
+        const appEntry = `import '${appPlugin}/phovea_registry.js';`;
+        assert.fileContent(`phovea_registry.js`, libEntry);
+        assert.fileContent(`phovea_registry.js`, appEntry);
+    });
+
+    it('generates workspace "package.json" with correct property "name"', () => {
+        assert.jsonFileContent('package.json', {name: pkg.name});
+    });
+
+    it('generates workspace "package.json" with correct property "version"', () => {
+        assert.jsonFileContent('package.json', {version: pkg.version});
+    });
+
+    it('generates workspace "package.json" with correct property "description"', () => {
+        assert.jsonFileContent('package.json', {description: pkg.description});
+    });
+
+    it('generates workspace "package.json" with `watch` property', () => {
+        assert.jsonFileContent('package.json', {watch: watch_content});
+    });
+
+    it('generates workspace "package.json" with the correct scripts', () => {
+        assert.jsonFileContent('package.json', {scripts: pkg.scripts});
+    });
+});
```

Comment:
Missing new line at the end of the file.

```suggestion
});

```